card/styles.css

```diff
@@ -14,17 +15,23 @@
   background-color: var(--color-card-background);
   border-radius: 0.25rem;
   line-height: 1.25;
-  max-width: 260px;
+  max-width: 400px;
   min-width: 260px;
 }
 .card-body {
   display: flex;
   flex-direction: column;
+  align-items: space-between;
   gap: 20px;
```

Comment:
Try to give the gaps and widths in rems too, to keep your values following a consistent ratio.

Reply:
Thank you will do .